### PR TITLE
cleanup edshipyard

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**
Standard treatment, newlines around scope changes, removed tabbed inline comments, replaced modulo formatting with .format formatting, replaced overcomplex oneliners

### CODE REVIEW

File: edshipyard.py [hunk 1]
```diff
@@ -40,21 +47,32 @@
...
 
 # Export ship loadout in E:D Shipyard plain text format
 def export(data, filename=None):
+    def class_rating(module: __Module):
+        mod_class = module['class']
+        mod_rating = module['rating']
+        mod_mount = module.get('mount')
+        mod_guidance = module.get('guidance')
+
+        ret = '{clazz}{rating}'.format(clazz=mod_class, rating=mod_rating)
```
Comment (Athanasius) [inline edshipyard.py:62]: Let's not get cute, use `module_class` to avoid the `class` keyword issue.

File: edshipyard.py [hunk 2]
```diff
@@ -69,92 +87,130 @@ def class_rating(module):
...
+
+    string += '---\nCargo : {} T\nFuel  : {} T\n'.format(cargo, fuel)
 
     # Add mass and range
     assert data['ship']['name'].lower() in companion.ship_map, data['ship']['name']
     assert companion.ship_map[data['ship']['name'].lower()] in ships, companion.ship_map[data['ship']['name'].lower()]
+
     try:
+        # TODO: broken link
         # https://github.com/cmmcleod/coriolis/blob/master/app/js/shipyard/module-shipyard.js#L184
```
Comment (Athanasius) [inline edshipyard.py:180]: I'm not even sure what information the URL would be conveying.  Just lose this line.

File: edshipyard.py [hunk 3]
```diff
@@ -69,92 +87,130 @@ def class_rating(module):
...
     if filename:
         with open(filename, 'wt') as h:
             h.write(string)
+
         return
 
     # Look for last ship of this type
     ship = companion.ship_file_name(data['ship'].get('shipName'), data['ship']['name'])
-    regexp = re.compile(re.escape(ship) + '\.\d\d\d\d\-\d\d\-\d\dT\d\d\.\d\d\.\d\d\.txt')
+    regexp = re.compile(re.escape(ship) + r'\.\d{4}-\d\d-\d\dT\d\d\.\d\d\.\d\d\.txt')
```
Comment (Athanasius) [inline edshipyard.py:203]: You cited the more compact version in Discord, didn't push the whole change ?

Comment (A-UNDERSCORE-D) [reply edshipyard.py:203]: I felt that the further compacting did not actually make it any more compact and increased noise, `\d{2}` is five characters while `\d\d` is four, and easier to parse

**Pull request merged**